src/main/java/org/spongepowered/api/data/Keys.java

```diff
@@ -1563,229 +1435,264 @@
 
     /**
-     * Represents the {@link Key} for the state that something is "lit",
-     * for example a {@link BlockTypes#FURNACE}, {@link BlockTypes#CAMPFIRE}
+     * Whether a {@link BlockState} is lit.
+     * e.g. {@link BlockTypes#FURNACE}, {@link BlockTypes#CAMPFIRE}
      * or {@link BlockTypes#REDSTONE_TORCH}.
      */
     public static final Supplier<Key<Value<Boolean>>> LIT = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "LIT");
 
     /**
-     * Represents the {@link Key} for a {@link Llama}s carrying strength. The higher the strength,
-     * the more items it can carry (effectively the size of inventory).
+     * A {@link Llama}'s {@link LlamaType}.
      */
-    public static final Supplier<Key<BoundedValue<Integer>>> LLAMA_STRENGTH = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "LLAMA_STRENGTH");
+    public static final Supplier<Key<Value<LlamaType>>> LLAMA_TYPE = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "LLAMA_TYPE");
 
     /**
-     * Represents the {@link Key} for a {@link Llama}'s {@link LlamaType}.
+     * The token used to lock a {@link CarrierBlockEntity}. Or the token on an {@link ItemStack} to unlock it.
      */
-    public static final Supplier<Key<Value<LlamaType>>> LLAMA_TYPE = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "LLAMA_TYPE");
+    public static final Supplier<Key<Value<String>>> LOCK_TOKEN = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "LOCK_TOKEN");
 
     /**
-     * Represents the {@link Key} for the token used to lock a
-     * {@link CarrierBlockEntity}.
+     * The displayed description ("lore") text of an {@link ItemStack}.
+     *
+     * <p>The lore text is usually displayed when the player hovers his cursor
+     * over the stack. For the contents of a book see {@link #PAGES}
+     * instead.</p>
      */
-    public static final Supplier<Key<Value<String>>> LOCK_TOKEN = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "LOCK_TOKEN");
+    public static final Supplier<Key<ListValue<Text>>> LORE = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "LORE");
 
     /**
-     * Represents the {@link Key} for the matter
-     * state. Usually of a block.
+     * The matter state of a {@link BlockState}
      */
     public static final Supplier<Key<Value<MatterState>>> MATTER_STATE = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "MATTER_STATE");
 
     /**
-     * Represents the {@link Key} for the maximum air supply a {@link Living}
-     * may have.
+     * The maximum amount of ticks between two
+     * batches of entities spawned by a {@link MobSpawner}.
+     */
+    public static final Supplier<Key<BoundedValue<Integer>>> MAXIMUM_DELAY = Sponge.getRegistry().getCatalogRegistry().provideSupplier(Key.class, "MAXIMUM_DELAY");
```

Comment:
I don't think this is specific enough, `MAX_SPAWN_DELAY`?
(`MAXIMUM` -> `MAX`) for consistency with other keys.